metricbeat/mb/module/event.go

```diff
@@ -68,37 +59,32 @@ func (b EventBuilder) Build() (common.MapStr, error) {
 	if n, ok := event["_namespace"]; ok {
 		delete(event, "_namespace")
 		namespace = n.(string)
```

Comment:
Just to be safe, this should check the boolean that is returned that indicates the success of the type assertion.